src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolder.java

```diff
@@ -1000,9 +1001,17 @@ public FolderIcon getIconColor() {
     @Exported(name="jobs")
     @Override
     public Collection<I> getItems() {
+        return getItems(item -> true);
+    }
+
+    /**
+     * Gets all the items in this collection in a read-only view that matches supplied Predicate
+     */
+    // TODO: @Override and inherit docs once baseline is above 2.222
```

Comment:
is not needed to update the baseline for this PR?

Reply:
We have to so that we can add the `@Override` annotation but this new function is only available in `2.222` so unless we are comfortable with pushing the baseline so high up for now we can add this code here and add the annotation later.